.zenodo.json

```diff
@@ -193,6 +201,7 @@
       "orcid": "0000-0002-9688-8950"
     },
     {
+      "affiliation": "Kitware Inc",
```

Comment:
Do we want to change names from ALL CAPS family names for lines https://github.com/thewtex/ITK/blob/bbf4a978c032ef21f5b78f34273e3976e7efd99f/.zenodo.json#L210-L212 and lines https://github.com/thewtex/ITK/blob/bbf4a978c032ef21f5b78f34273e3976e7efd99f/.zenodo.json#L222-L225?

Reply:
No, these are the author's preference.